### PR TITLE
[Android migration page] remove the redundant info 

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: src/pages/documentation/migrate-to-android.md [hunk 1]
```diff
@@ -79,11 +79,12 @@ Kotlin
 
 Some of the APIs available in previous major versions of the Mobile SDK for Android are now deprecated. You can choose to replace the deprecated APIs in your code with the alternative APIs in the latest version, as described in the table below:
 
+<InlineAlert variant="info" slots="text"/>
+
+The `registerExtension` API for each extension is deprated in the latest mobile SDK. You don't need to register extensions seperately, now you can call `MobileCore.registerExtensions` instead. See [Update SDK initialization](#update-sdk-initialization) section for more details.
```
Comment (calebk1) [inline src/pages/documentation/migrate-to-android.md:84]: "See the [update SDK inutialization] section..."

**Review by calebk1 — APPROVED** [review]
Looks good! Just one minor comment.

File: src/pages/documentation/migrate-to-android.md [hunk 2]
```diff
@@ -79,11 +79,12 @@ Kotlin
 
 Some of the APIs available in previous major versions of the Mobile SDK for Android are now deprecated. You can choose to replace the deprecated APIs in your code with the alternative APIs in the latest version, as described in the table below:
 
+<InlineAlert variant="info" slots="text"/>
+
+The `registerExtension` API for each extension is deprated in the latest mobile SDK. You don't need to register extensions seperately, now you can call `MobileCore.registerExtensions` instead. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
```
Comment (swarna04) [inline src/pages/documentation/migrate-to-android.md:84]: typo: deprated -> deprecated, seperately-> separately

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: src/pages/documentation/migrate-to-android.md (hunk 2 above)
Comment (calebk1) [reply src/pages/documentation/migrate-to-android.md:84]: Oh, and one more - "... in the latest version of the Mobile SDK."

**Review by emdobrin — APPROVED** [review]

**Pull request merged**